aether-ui/aether/ui/assets/apps/pipeline/components/DeleteStatus.jsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { Component } from 'react'
+import { Modal } from '../../components'
+import { connect } from 'react-redux'
+import { FormattedMessage } from 'react-intl'
+
+class DeleteStatus extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+    }
+  }
+
+  componentDidUpdate () {
+    if (this.props.error || (this.props.deleteStatus &&
+      this.props.deleteStatus.not_published)) {
+      this.props.toggle()
+    }
+  }
+
+  render () {
+    if (!this.props.showModal) {
+      return null
+    }
+
+    const header = (
+      <span>
+        <FormattedMessage
+          id='delete.progress.modal.header'
+          defaultMessage={this.props.header}
```

Comment:
:disappointed:

Reply:
In this case, this is useless and creates more confusion than me!!! :stuck_out_tongue: 
The simple `const header = this.props.header` is enough. You already translated it. :gb: :nigeria: :de:

Reply:
Ahhhh... yes, true. The back and forth is throwing me off...  😄